rust/src/lib.rs

```diff
@@ -208,11 +208,12 @@ pub struct TransactionBody {
     inputs: TransactionInputs,
     outputs: TransactionOutputs,
     fee: Coin,
-    ttl: u32,
+    ttl: Option<Slot>,
```

Comment:
Are TTL and validity start intervals the only places we use absolute slots and the others are relative or how is it? We should probably have two type aliases here to be clear even if they're stripped out in the wasm builds.

Reply:
I believe Timelock Start and Timelock Expire are both absolute slots as well. Additionally, `HeaderBody` also has this.